tests/unit/types/test_exceptions.py

```diff
@@ -57,6 +57,25 @@
                 "locations": [{"line": 2, "column": 2}],
             },
         ),
+        (
+            "Init GraphQLError",
+            {
+                "path": ["init", "path"],
+                "locations": [Location(line=1, column=1)],
+                "user_message": "User GraphQLError",
```

Comment:
I would suggest to simplify this test for better readability: let's just test for the `extensions` key effects.

Reply:
I usually like iterative tests where we add optional things as we go during the test suite so I indeed added the extensions key to the previous test input.

Your call guys, just tell me and I'll update this?

Reply:
I think adding usecases to the test is enough here.

But i'd like to see a 'functional' test that validate this feature.
It could be added to the functionnal/regression suite, for no-regression purpose, in a file named test_pr155.py.

Reply:
done @abusi